examples/simultaneous_translation/criterions/cross_entropy_acc.py

```diff
@@ -13,122 +13,6 @@
 from fairseq import utils
 from fairseq.criterions import FairseqCriterion, register_criterion
 
-
-@register_criterion("cross_entropy_acc")
-class CrossEntropyWithAccCriterion(FairseqCriterion):
```

Comment:
Why remove this?

Reply:
Now I import this from speech recognition repo so this will raise a duplicated registration error